Notes.txt

```diff
@@ -0,0 +1,12 @@
+- Where do we put projects that are associates/tier 2?  Kubernetes puts this in a separate org.  Do we just want to have a certain naming convention? Add the instructions to the odh-repositories md file.
```

Comment:
I can remove this file for now.  It was intended to track considerations we need to answer as a community, but this is best left to an issue instead of a file.